devimint/src/tests.rs

```diff
@@ -2132,6 +2133,7 @@ pub async fn recoverytool_test(dev_fed: DevFed) -> Result<()> {
         .iter()
         .map(|o| o["descriptor"].as_str().unwrap())
         .collect::<HashSet<_>>();
+    info!(?epochs_descriptors);
```

Comment:
If you made this `debug`, have a proper message, `target`, etc then it wouldn't even be "hack", but just "improved logging".